test/performance/odp_l2fwd.c

```diff
@@ -41,6 +41,12 @@
 /* Maximum pktio index table size */
 #define MAX_PKTIO_INDEXES      1024
 
+/* Default vector size */
+#define DEFAULT_VEC_SIZE       4
```

Comment:
This default value is rather small and causes poor performance on linux-generic. I'd suggest setting `DEFAULT_VEC_SIZE `to `MAX_PKT_BURST`.

Reply:
Makes sense. Will change to `MAX_PKT_BURST`.